# U.S. Medical Insurance Costs

The purpose of this project is to investigate a U.S. medical insurance dataset for variables that are likely to affect a person's insurance costs. Additionally, each of the variables being considered will be analysed for tendencies in the dataset. Variables will also be compared with each other to determine if there is a relationship between any of them.

## Data Overview

### Importing Dataset

In [17]:
import csv

with open('insurance.csv', newline = '') as insurance_csv:
    reader = csv.DictReader(insurance_csv, delimiter = ',')
    data = list(reader)

### Data Cleaning

In [19]:
# Converting numerical values from string to integer and float values (age, sex, bmi, children, charges)


### Variables

In [18]:
print("Number of patients in the dataset: {}".format(len(data)))
print("Variables in the dataset: {}".format(data[0].keys()))

Number of patients in the dataset: 1338
Variables in the dataset: dict_keys(['age', 'sex', 'bmi', 'children', 'smoker', 'region', 'charges'])


The medical insurance cost dataset contains characteristics for 1,338 patients. For each of the patients the dataset measures 7 different characteristics:

- Age
- Sex
- BMI
- Number of Children
- Smoker or Non-Smoker
- Region
- Medical Insurance Charges


Out of the 7 variables being measured the dependent variable in this project will be the medical insurance charges while the remaining 6 variables will be considered the independent variables.

## Analysis

### Descriptive Statistics